helm/cosmo/charts/router/templates/deployment.yaml

```diff
@@ -123,6 +128,14 @@ spec:
                   name: {{ include "router.secretName" . }}
                   key: graphApiToken
             {{- end }}
+            - name: PROMETHEUS_ENABLED
```

Comment:
Please write all values into the configmap first and then reference them here.

Reply:
Please let me know if it is fixed now